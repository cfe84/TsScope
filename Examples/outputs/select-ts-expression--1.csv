"id","header_1","header_2","thisOneIsAliased","number","aBool"
"ID-1","JOHN","Yop","Yop",54,true
"ID-2","JANE","Yop","Yop",54,true
"ID-3","JIM","Yop","Yop",54,true
"ID-4","JILL","Yop","Yop",54,true
"ID-5","JACK","Yop","Yop",54,true
"ID-6","JOYCE","Yop","Yop",54,true
"ID-7","JAMES","Yop","Yop",54,true
"ID-8","JENNIFER","Yop","Yop",54,true
"ID-9","JERRY","Yop","Yop",54,true
"ID-10","JUDY","Yop","Yop",54,true
"ID-11","RICHIS","Yop","Yop",54,true
"ID-12","MILLA","Yop","Yop",54,true
"ID-13","BOB","Yop","Yop",54,true
