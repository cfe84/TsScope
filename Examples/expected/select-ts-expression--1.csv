"id,"field_1,"field_2,"thisOneIsAliased,"number,"aBool
"ID-1","JOHN","Yop","Yop",54,true
"ID-2","JANE","Yop","Yop",54,true
"ID-3","JIM","Yop","Yop",54,true
"ID-4","JILL","Yop","Yop",54,true
"ID-5","JACK","Yop","Yop",54,true
"ID-6","JOYCE","Yop","Yop",54,true
"ID-7","JAMES","Yop","Yop",54,true
"ID-8","JENNIFER","Yop","Yop",54,true
"ID-9","JERRY","Yop","Yop",54,true
"ID-10","JUDY","Yop","Yop",54,true
